ethereum/core/src/main/java/org/hyperledger/besu/ethereum/core/Transaction.java

```diff
@@ -753,6 +768,7 @@ private static Bytes32 computeSenderRecoveryHash(
         preimage = frontierPreimage(nonce, gasPrice, gasLimit, to, value, payload, chainId);
         break;
       case EIP1559:
+      case BLOB:
```

Comment:
Here we need to implement a dedicated `blobPreimage` method that takes in consideration the new field, as I am doing for the `max_fee_per_data_gas` field.
If you want I can it in my PR for both fields.

Reply:
perfect, thanks.